pkg/asset/ignition/bootstrap_ignition.go

```diff
@@ -222,3 +245,72 @@ func (i *BootstrapIgnition) addExtraManifests(config *igntypes.Config, extraMani
 
 	return nil
 }
+
+func (i *BootstrapIgnition) addICSPManifests(envConfig *config.EnvConfig) error {
+	// Read ImageContentSourcePolicy manifests
+	icspListBytes, err := os.ReadFile(filepath.Join(envConfig.CacheDir, icspFileName))
```

Comment:
note: if we will want to as UTs here, the os.readfile should by inserted by in interface here. 
We have examples for it on other UTs for pkgs